Cargo.toml

```diff
@@ -9,6 +9,8 @@ members = [
   "identity-did",
   "identity-diff",
   "identity-iota",
+  "identity-iota-core",
+  "identity-account-core",
```

Comment:
This crate seems 95% storage-related, so can we rename this to `identity-account-storage`, or preferably, even just `identity-storage`?

Reply:
It originally was `identity-account-storage` but changed after several types there too. If all those types are related to the `Storage` I would agree with the name change then. I wouldn't choose just `identity-storage` since it really is tied to the `Account`, it's not usable/useful without it.

Reply:
To me, it seems like almost all of it is storage-related, so the name seems fine. If we can be more specific than `core`, I think we should do it. Happy with `identity-account-storage`, too!